src/SFA.DAS.EAS.Portal.Client/Data/DocumentClientFactory.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using SFA.DAS.CosmosDb;
+
+namespace SFA.DAS.EAS.Portal.Client.Data
+{
+    public class DocumentClientFactory : IDocumentClientFactory
+    {
+        private readonly ICosmosDbConfiguration _cosmosDbConfiguration;
+
+        public DocumentClientFactory(ICosmosDbConfiguration cosmosDbConfiguration)
+        {
+            _cosmosDbConfiguration = cosmosDbConfiguration;
+        }
+
+        public IDocumentClient CreateDocumentClient()
+        {
+            var connectionPolicy = new ConnectionPolicy
+            {
+                RetryOptions =
+                {
+                    MaxRetryAttemptsOnThrottledRequests = 2,
```

Comment:
Need to put these in config.

Reply:
Do these values _need_ to be configurable? If we can come up with sensible values that are unlikely to change, keeping the number of config values down leads to simpler configuration. These values are the same as used by existing DAS Cosmos DB clients, which haven't had to be changed so far.

If we decide to make them configurable, I would suggest we allow the consumer of the client to supply the values, which they can either hard-code or read from their own config as appropriate.